### ISSUE TITLE
Output format

### ISSUE THREAD

**Issue by YellowManPL** [vivekkalyanarangan30/searchengine#2]
Hi vivek,

I guess evaluating output from .json form to more 'clean' one will be nice feature. 
Basically: removing [] and ""

**Comment by vivekkalyanarangan30** [comment]
Yes @YelloManPL I am taking care of that through the front end. The cleaner approach makes more sense though. On to it...
